### ISSUE TITLE
Feature Request: Configurable Source Sever Url

### ISSUE THREAD

**Issue by Inspyro** [re-motion/BuildTools#2]
It would be a cool feature to add a possibility to configure the source server url on the client (see: _InsertSourceLinks.cs_)

So probably use a system environment variable, or a config file, so that the user can override / change the hard coded url in the pdb file. 
It would be necessary to adjust the powershell snippet that is inserted by the InsertSourceLinks task into the pdb files, so that it checks for config file / environment variable.

**Comment by MichaelKetting** [comment]
Yes, there's a concept for this available in the vcs-data definition. PRs welcome :)
